### PR TITLE
Add docs for tf.strings.lower and tf.strings.regex_replace

### PR DESCRIPTION
**Pull request by lc0**
Funny enough it even showed as TODO in docs and on the website
image

### CODE REVIEW

File: tensorflow/core/api_def/base_api/api_def_StringLower.pbtxt [hunk 1]
```diff
@@ -1,3 +1,11 @@
 op {
   graph_op_name: "StringLower"
+  summary: "Converts all uppercase characters into their respective lowercase replacements."
+  description: <<END
+Example:
+```
```
Comment (mihaimaruseac) [inline tensorflow/core/api_def/base_api/api_def_StringLower.pbtxt]: No need for the ` ``` ` lines, just leave them empty, please.

**Review by mihaimaruseac — DISMISSED** [review]

File: tensorflow/core/api_def/base_api/api_def_StringLower.pbtxt (hunk 1 above)
Comment (lc0) [reply tensorflow/core/api_def/base_api/api_def_StringLower.pbtxt]: Done, was nor sure how description is going to be propagated from `pbtxt`

**Comment by lc0** [comment]
Fixed line split, thanks for force-run!

**Comment by lc0** [comment]
Windows build failing is not related to introduced change. cc @mihaimaruseac 

Does it make sense not even to suppress this part? 
Feels like it's pretty common :)

**Comment by mihaimaruseac** [comment]
It is fixed by adc3a1b1bec28849f62076e9b4be5c5963e5e5e7 so after a rebase on master it should work.

But in any case, this is blocked now on copybara and internal import, nothing needed for you to do at the moment

**Review by mihaimaruseac — DISMISSED** [review]

**Comment by yifeif** [comment]
Sorry for the delay, do you mind resolving the conflicts @lc0 ?

**Comment by lc0** [comment]
@yifeif just rebased and resolved conflicts

**Review by mihaimaruseac — APPROVED** [review]

**Pull request merged**